dart/tool/service_undocumented.md

```diff
@@ -17,29 +17,6 @@ Success _requestHeapSnapshot(string isolateId, string roots, bool collectGarbage
 _roots_ is one of User or VM. The results are returned as a stream of
 [_Graph] events.
 
-### _getAllocationProfile
-
-```
-AllocationProfile _getAllocationProfile(
-  string isolateId,
-  string gc [optional],
-  bool reset [optional]
-)
-```
-
-Valid values for _gc_ are 'full'.
-
-
-### _getInstances
-
-```
-@Object _getInstances(
-  string isolateId,
-  string classId,
-  int limit
-)
-```
-
 Returns a ServiceObject (a specialization of an ObjRef).
```

Comment:
This line can be deleted (it goes with the proceeding declaration).

Reply:
deleted.